youtube_dl/extractor/bandcamp.py

```diff
@@ -47,6 +47,15 @@ def _real_extract(self, url):
         mobj = re.match(self._VALID_URL, url)
         title = mobj.group('title')
         webpage = self._download_webpage(url, title)
+        album = self._search_regex(
+            r'(?ms).*?title\s*?:\s*?"(?P<album>.*?)",',
```

Comment:
So, something like this?
```
album = self._search_regex(
    r'(?ms).*?album_title\s*?:\s*?"(?P<album>[^"]+)",',
```
Also, if a track is just a single track and not part of an album (such as the youtube-dl test song), do we just want to leave album as NA or have the album name be the same as the track title?  (Perhaps that's just a personal preference of some?)